certbot-apache/certbot_apache/http_01.py

```diff
@@ -172,4 +172,10 @@ def _set_up_include_directives(self, vhost):
             self.configurator.parser.add_dir(
                 vhost.path, "Include", self.challenge_conf_post)
 
+            if not vhost.enabled:
+                from certbot_apache.parser import get_aug_path
+                self.configurator.parser.add_dir(
```

Comment:
While not directly related to this PR, is the reason you used this rather than calling `self.configurator.parser.add_include` because we don't want to add this path to the `existing_paths` list in the parser? If so, is our usage of it in TLS-SNI a bug?

Reply:
Yes, not adding the path to `existing_paths` is the reason. And you are right, it is a bug in our TLS-SNI validation. However I'm pretty confident that it doesn't manifest itself in any way in case of TLS-SNI, as the validation `ServerName` is very unlikely matching anything we search.